### PR TITLE
Release profiling tool jar to maven central

### PR DESCRIPTION
**Pull request by NvTimLiu**
Release profiling tool jar to maven central along with 21.06 rapids-plugin jars

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
fixes issue https://github.com/NVIDIA/spark-rapids/issues/2556

File: rapids-4-spark-tools/pom.xml [hunk 1]
```diff
@@ -151,6 +151,10 @@
                 <groupId>org.apache.rat</groupId>
                 <artifactId>apache-rat-plugin</artifactId>
 	    </plugin>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
```
Comment (NvTimLiu) [inline rapids-4-spark-tools/pom.xml:156]: Add scala-maven-plugin to generate scala javadoc for the release deploy

**Comment by pxLi** [comment]
looks like profiling tool artifactID does not have the scala_binary version as suffix, 
not sure if we would like to add it `rapids-4-spark-tools` -> `rapids-4-spark-tools_2.12`

**Comment by tgravescs** [comment]
yes it should have the scala version in it

**Comment by NvTimLiu** [comment]
Keep the `module` to `rapids-4-spark-tools`, and add scala_binary version sufix to the profiling tool artifactID: `rapids-4-spark-tools_2.12`

the files `rapids-4-spark-tools/src/test/resources/XXX_eventlog` should be generated automatically so we can not change it?

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Pull request merged**